<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KResizeDialog</class>
 <widget class="QDialog" name="KResizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>131</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Resize</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <number>6000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_2">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="maximum">
        <number>6000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Width :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBox">
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>Keep ratio</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="toolTip">
        <string>Resize the image with seamCarving algorithm</string>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>Smart Resize</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDialogButtonBox" name="buttonBox_2">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>KResizeDialog</receiver>
   <slot>changeHeightValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>KResizeDialog</receiver>
   <slot>changeWidthValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>KResizeDialog</receiver>
   <slot>validateResize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>KResizeDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeHeightValue()</slot>
  <slot>changeWidthValue()</slot>
  <slot>validateResize()</slot>
 </slots>
</ui>
